packages/core/src/components/HelperAndErrorTextTooltip/HelperAndErrorTextTooltip.stories.mdx

```diff
@@ -6,14 +6,17 @@ import { Custom } from './HelperAndErrorTextTooltip.stories.tsx';
 
 # HelperAndErrorTextTooltip
 
-HelperAndErrorTextTooltip component
+The `HelperAndErrorTextTooltip` displays informative text when you hover over or tap an element. When activated, it displays a text label identifying an element, such as a description of its function.
```

Comment:
We can simply keep `The HelperAndErrorTextTooltip displays informative text when you hover over or tap an element.`